app/package.json

```diff
@@ -2,21 +2,30 @@
   "private": true,
   "scripts": {
     "dev": "nodemon --watch backend server NODE_OPTIONS='--inspect'",
-    "build": "next build",
-    "start": "node server",
+    "build": "yarn run build:server && yarn build:next",
+    "build:next": "next build",
+    "build:server": "tsc --project tsconfig.server.json",
```

Comment:
looks like we're missing a `build:schema` target?
https://github.com/bcgov/cas-cif/blob/develop/app/package.json#L14